tools/section-link-check.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+for file in `find . -name "*.md" -type f`; do
+  echo "Checking $file..."
+  sections=$(pandoc $file --verbose --fail-if-warnings -o pdfs/tmp.pdf 2>&1 | grep -E 'pdfTeX warning \(dest\): name{[^}]+}' | sed -E  's/.*name\{([^}]+)\}.*/\1/' | sort | uniq)
```

Comment:
1. Please add a comment saying that you are `Extracting all broken hyperref detected by PDFTex`.
2. Broken refs can be anything, not just sections, so I think that  the name of the variable should be `s/sections/broken_refs/`